src/utils/GetTestResults.ts

```diff
@@ -12,13 +12,6 @@ export class GetTestResults {
     });
   }
 
-  public static filterTestResultByDate(testResults: any, fromDateTime: string | number | Date, toDateTime: string | number | Date) {
-
-    return testResults.filter((testResult: { testStartTimestamp: string | number | Date; testEndTimestamp: string | number | Date; }) => {
-      return moment(testResult.testStartTimestamp).isAfter(fromDateTime) && moment(testResult.testEndTimestamp).isBefore(toDateTime);
```

Comment:
Tidy the imports. moment is no longer used.

Reply:
The changes requested were minor so they have been pushed to 18782 to reduce rebases.